src/pages/index.tsx

```diff
@@ -62,16 +66,65 @@ const IndexPage: NextPage = () => {
      */
   };
 
+  const isLocalHost = global.location.href.includes("localhost");
+
   return (
     <Box padding={8}>
       <Text variant={"hero"}>Welcome to Saleor App Template (Next.js) 🚀</Text>
       <Text as={"p"} marginTop={8}>
-        This is a boilerplate you can start with, to create an app connected to Saleor
+        Explore the App Template by visiting:
       </Text>
+      <ul>
+        <li>
+          <code>src/pages/api/manifest</code> - the{" "}
+          <a
+            href="https://docs.saleor.io/docs/3.x/developer/extending/apps/manifest"
+            target="_blank"
+            rel="noreferrer"
+          >
+            App Manifest
+          </a>
+          .
+        </li>
+        <li>
+          <code>src/pages/api/webhooks/order-created</code> - an example <code>ORDER_CREATED</code>{" "}
+          webhook handler.
+        </li>
+        <li>
+          <code>graphql</code> - the pre-defined GraphQL queries.
```

Comment:
maybe lets mark it more as a directory. `/graphql` or a simple icon. Other paths the same